source/blog/2018-08-27-ember-3-4-released.md

```diff
@@ -64,6 +64,43 @@ Guides will be updated to reflect the new syntax in the coming weeks.
 
 Custom component manager (2 of 2)
 
+The second new feature of Ember 3.4 is the custom component manager. This API will allow addon authors to provide special-purpose component base classes that their users can subclass from in apps. These components are invokable in templates just like any other Ember components (descendants of `Ember.Component`) today.
+
+Component managers are registered with the `component-manger` type in the application's registry. Similar to services, component managers are singleton objects (i.e. `{ singleton: true, instantiate: true }`), meaning that Ember will create and maintain (at most) one instance of each unique component manager for every application instance.
+
+To register a component manager, an addon will put it inside its `app` tree:
+```js
+// ember-basic-component/app/component-managers/basic.js
+
+import EmberObject from '@ember/object';
+
+export default EmberObject.extend({
+  // ...
+});
+```
+This allows the component manager to participate in the DI system – receiving injections, using services, etc. Alternatively, component managers can also be registered with imperative API. This could be useful for testing or opt-ing out of the DI system. For example:
```

Comment:
do you think it's useful to mention the imperative API in the release blog post? I think it would go in the documentation.